src/content/en/2021/seo.md

```diff
@@ -1,20 +1,723 @@
+
+The Web Almanac 2020 Report noted that "A similar conflict can be found with the different implementation methods, with 0.15% of the mobile pages and 0.17% of the desktop ones showing conflicts between the canonical tags implemented via their HTTP headers and HTML head."
+
+This year's data is even more worrisome. Pages are sending conflicting signals in 0.4% of cases.
+
+As the Web Almanac data relies on homepages, there may be additional problems with pages located deeper in the architecture. The data also doesn't include situations where more than one canonical tag appears with a single method.
+
+## Page Experience
+
+2021 saw an increased focus on user experience. Google launched the Page Experience Update which included existing signals, such as HTTPS and mobile friendliness, and new speed metrics called Core Web Vitals.
+
+### HTTPS
+
+Adoption of HTTPS is still increasing. HTTPS was the default on 81.17% of mobile pages and 84.29% of desktop pages. That's up nearly 8% on mobile websites and 7% on Desktop websites year over year.
+
+{{ figure_markup(
+   image="usage-of-https.png",
+   caption="Percentage of Desktop and Mobile pages served with HTTPS..",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1826599611&format=interactive",
+   sheets_gid="1347655296",
+   sql_file="seo-stats.sql"
+   )
+}}
+
+### Mobile-Friendliness
+
+There's a slight uptick in mobile-friendliness this year. Responsive design implementations have increased while dynamic serving has remained relatively flat.
+
+Responsive design sends the same code and adjusts how the website is displayed based on the screen size, while dynamic serving will send different code depending on the device. The viewport meta tag was used to identify responsive websites vs the Vary: User-Agent header to identify websites using dynamic serving.
+
+ 91.06% of mobile pages include the viewport meta tag, up from 89.16% in 2020. 86.44% of desktop pages also included the viewport meta tag, up from 83.82% in 2020.
+
+{{ figure_markup(
+   image="vary-usage-agent-header-usage.png",
+   caption="`Vary: User-Agent` header usage.",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1985736287&format=interactive",
+   sheets_gid="478009067",
+   sql_file="html-response-vary-header-used.sql"
+   )
+}}
+
+For the Vary: User-Agent header, the numbers were pretty much unchanged with 12.6% of desktop pages and 13.4% of mobile pages with this footprint.
+
+One of the biggest reasons for failing mobile-friendliness was that 13.5% of pages did not use a legible font size. Meaning 60% or more of the text had a font size smaller than 12px which can be hard to read on mobile.
```

Comment:
Isn't the threshold size 16px?

Reply:
The check here is defined this way, see [https://web.dev/font-size/](https://web.dev/font-size/).

Reply:
Add the link to the text to make this clear.